licence-connect-authenticator/src/main/resources/META-INF/licence-changelog.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
+
+    <include file="META-INF/licence-changelog-1.0.xml"/>
```

Comment:
I like it better to name the file after its content/purpose: `add-licence-table.xml` 

and no version please